src/SFA.DAS.RoatpOversight.Web/Infrastructure/ApiClients/ApplyApiClient.cs

```diff
@@ -28,7 +28,7 @@ public class ApplyApiClient : ApiClientBase<ApplyApiClient>, IApplyApiClient
 
         public async Task Ping()
         {
-            await Get($"/ping");
+            await Get<string>($"/ping");
```

Comment:
Did this creep in accidentally? I can't remember if you need to use the generic end point or not

Reply:
No, it's still needed as the client is used to call the ping endpoint. The package change has meant that the non-generic method has been removed hence this change.